modules/ingest-geoip/build.gradle

```diff
@@ -28,8 +28,8 @@ dependencies {
   // Upgrade to 2.10.0 or higher when jackson-core gets upgraded to 2.9.x. Blocked by #27032
   compile('com.maxmind.geoip2:geoip2:2.9.0')
   // geoip2 dependencies:
-  compile("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
-  compile("com.fasterxml.jackson.core:jackson-databind:${versions.jacksondatabind}")
+  compile("com.fasterxml.jackson.core:jackson-annotations:2.8.11")
+  compile("com.fasterxml.jackson.core:jackson-databind:2.8.11.6")
```

Comment:
Should these still use the `${versions.jackson}` (etc) substitution instead of being hardcoded?

Reply:
I explained in the commit message why I made this change. See also #53523.

Reply:
Doh, sorry, yes I totally forgot to read that.